src/components/navBar/index.tsx

```diff
@@ -1,17 +1,65 @@
 import Image from 'next/image';
+import Link from 'next/link';
+import { LOGIN_URL, USER_PROFILE_URL, DEFAULT_AVATAR } from '@/components/constants/url'
+import useAuthenticated from '@/hooks/useAuthenticated';
 import styles from '@/components/navBar/navBar.module.scss';
-
 const RDSLogo = '/RDSLogo.png';
 
-const NavBar = () => (
-  <nav className={styles.navBar}>
-    <a className={styles.logo} href="https://realdevsquad.com"><Image width="45px" height="45px" src={RDSLogo} alt="real-dev squad" /></a>
-    <a href="https://welcome.realdevsquad.com/">Welcome</a>
-    <a href="https://www.realdevsquad.com/events.html">Events</a>
-    <a href="https://members.realdevsquad.com/">Members</a>
-    <a href="https://crypto.realdevsquad.com/">Crypto</a>
-    <a className={styles.active} href="https://status.realdevsquad.com/">Status</a>
-  </nav>
-);
+const NavBar = () => {
+  const GITHUB_LOGO = '/github-white.png';
+  const { userData, isLoggedIn } = useAuthenticated()
+
+  return (
+    <nav className={styles.navBar}>
+      <div>
+        <a className={styles.logo} href="https://realdevsquad.com"><Image width="45px" height="45px" src={RDSLogo} alt="real-dev squad" /></a>
+        <a href="https://welcome.realdevsquad.com/">Welcome</a>
+        <a href="https://www.realdevsquad.com/events.html">Events</a>
+        <a href="https://members.realdevsquad.com/">Members</a>
+        <a href="https://crypto.realdevsquad.com/">Crypto</a>
+        <a className={styles.active} href="https://status.realdevsquad.com/">Status</a>
+      </div>
+      <div>
+        {!isLoggedIn ? (
+          <Link href={LOGIN_URL}>
+            <a className={styles.btnLogin}>
```

Comment:
Could you put className to `Link` tag? Having `Link` >`a` >`button` doesn't make sense. Let's remove `a`. What do you think?

Reply:
https://nextjs.org/docs/api-reference/next/link I referred to it to do so and the button I have used inside it because of the accessibility